mediafile.py

```diff
@@ -1643,6 +1643,12 @@ def update(self, dict):
         StorageStyle('ARTIST'),
         ASFStorageStyle('Author'),
     )
+    artists = ListMediaField(
+        # MP3ListDescStorageStyle(desc='ARTISTS'),
```

Comment:
It looks like this field doesn't yet have an MP3 mapping?

Reply:
You mean I'd need to code separate class for this?

Reply:
Hmm, I wasn't attempting to suggest that. I was just trying to clarify: it looks like the MP3 mapping is disabled for this tag? Is that intentional? Is there an MP3 equivalent to this tag that we *should* be considering, and if so, what do we need to do to support it?

Reply:
There's no class like that, it's placeholder ATM.
More at my own review above (I already tagged you).